### PR TITLE
AbramovD he1 t2

### PR DESCRIPTION
**Pull request by DenisAbramovDm**
I finished my second homework on English

### CODE REVIEW

**Review by LiudmilaZhdanok — APPROVED** [review]
Lgfm

**Review by AnastasiaSheremet — DISMISSED** [review]

File: homework/english/denis_abramov/lecture1/vocabulary.md [hunk 1]
```diff
@@ -2,3 +2,17 @@ Today my computer is booting so slowly.
 In creative mode you can create what you want.
 Every day my computer connects to network automatically.
 I need to debug this programm.
+I often delete unnecessary files from my computer.
+I'm developing a chat-bot at this month.
+Do you disable plugins in browser, in wich you have no need.
+Don't forget to disconnect all cables before opening electrical panel.
+This interface doesn't display my new feature.
+He always enables VPN when his looking through Nexta channels.
```
Comment (Okkss) [inline homework/english/denis_abramov/lecture1/vocabulary.md:10]: Better use “turn on”, the VPN, not looking ( maybe view)

File: homework/english/denis_abramov/lecture1/vocabulary.md [hunk 2]
```diff
@@ -2,3 +2,17 @@ Today my computer is booting so slowly.
...
+Your command cannot been executed.
+Implementing new features in your project is good practice.
+I was constantly installing and uninstalling different programs at my recent work.
```
Comment (Okkss) [inline homework/english/denis_abramov/lecture1/vocabulary.md:13]: 12. Present Simple 
13. Constantly ( markers Present Simple)

File: homework/english/denis_abramov/lecture1/vocabulary.md [hunk 3]
```diff
@@ -2,3 +2,17 @@ Today my computer is booting so slowly.
...
+U need to plug in a keyboard to make the process of programming easyer.
+He cant restore his files, because he had pressed shift+del.
+I try to update drivers as often as possible.
+Your computer is too old. You have to upgrade it.
```
Comment (Okkss) [inline homework/english/denis_abramov/lecture1/vocabulary.md:17]: Not correct

File: homework/english/denis_abramov/lecture1/vocabulary.md [hunk 4]
```diff
@@ -2,3 +2,17 @@ Today my computer is booting so slowly.
 In creative mode you can create what you want.
 Every day my computer connects to network automatically.
 I need to debug this programm.
+I often delete unnecessary files from my computer.
+I'm developing a chat-bot at this month.
+Do you disable plugins in browser, in wich you have no need.
+Don't forget to disconnect all cables before opening electrical panel.
+This interface doesn't display my new feature.
+He always enables VPN when his looking through Nexta channels.
+Your command cannot been executed.
```
Comment (Okkss) [inline homework/english/denis_abramov/lecture1/vocabulary.md:11]: Negative form

**Review by AnastasiaSheremet — APPROVED** [review]

**Review by LiudmilaZhdanok — APPROVED** [review]
Lgfm

**Review by Alexandr-R-94 — APPROVED** [review]

**Pull request merged**